ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -282,4 +305,11 @@ hsConDeclsBinders cons
              -> [LFieldOcc GhcPs]
     get_flds flds = concatMap (cd_fld_names . unLoc) (unLoc flds)
 
-
+getLocalBindings :: RefMap Type -> LHsDecl GhcPs -> [Name]
```

Comment:
This could do with some explanation: I'm not sure reading this why this is the right way to get the local bindings.

Reply:
What is confusing? Is it the use of `getFuzzyScope` or is it something else?

Reply:
I guess I'm just rather unsure why this is the right thing to do? Why is it okay to use `getFuzzyScope` rather than `getLocalScope`? What's going on with position mapping?

Reply:
According to my testing, `getLocalScope` only returns symbols at the current start point (the start of the given function), which does not appear to have the locals in scope. `getFuzzyScope` returns all symbols that have intersecting scopes. This makes sense to me when I look at the docs for `IntervalMap.FingerTree`'s `dominator` (`getLocalScope`) and `intersections` (`getFuzzyScope`):

`dominators`:
> O(k log (n/k)). All intervals that contain the given interval, in lexicographical order.

`intersections`
>O(k log (n/k)). All intervals that intersect with the given interval, in lexicographical order.

It's clear that there at least needs a comment there, but maybe it would be better to create a new function with a clearer name?

Reply:
> According to my testing, getLocalScope only returns symbols at the current start point (the start of the given function), which does not appear to have the locals in scope.

I feel confused about what `getLocalScope` is for then, if not this? Are we using it wrong?